### PR TITLE
docs: Add mermaid version to script URL

### PR DESCRIPTION
**Pull request by sidharthv96**
## :bookmark_tabs: Summary

Version was absent in URL.

- Related https://github.com/mermaid-js/mermaid/issues/3590#issuecomment-1271222489

## :straight_ruler: Design Decisions

Describe the way your implementation works or what design decisions you made if applicable.

### :clipboard: Tasks

Make sure you

- [ ] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [ ] :bookmark: targeted `develop` branch

### CODE REVIEW

File: packages/mermaid/src/docs.mts [hunk 1]
```diff
@@ -41,6 +41,8 @@ import { remark } from 'remark';
 // @ts-ignore No typescript declaration file
 import flatmap from 'unist-util-flatmap';
 
+const version = JSON.parse(readFileSync('package.json', 'utf8')).version;
```
Comment (sidharthv96) [inline packages/mermaid/src/docs.mts:44]: Wondering if we should use this version or the latest version from npm/jsdelivr?

Comment (aloisklink) [reply packages/mermaid/src/docs.mts:44]: Should we just hardcode the version to:
`https://cdn.jsdelivr.net/npm/mermaid@9/dist/mermaid.min.js`?

Right now it redirects to v9.1.7, and it should automatically redirect to the latest `^9.0.0` version on jsdelivr.

File: vdocs/intro/index.md [hunk 2]
```diff
@@ -265,7 +265,7 @@ To Deploy Mermaid:
 **To deploy mermaid without a bundler, one can insert a `script` tag with an absolute address and a `mermaidAPI` call into the HTML like so:**
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/mermaid@<MERMAID_VERSION>/dist/mermaid.min.js"></script>
 <script>
   mermaid.initialize({ startOnLoad: true });
 </script>
```
Comment (aloisklink) [inline vdocs/intro/index.md:271]: Should we recommend using ESM instead? If we're planning on moving to ESM only at some point in the future, we might as well let people get used to using ESM.

```suggestion
<script type="module">
  import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@9/dist/mermaid.esm.min.mjs';
  mermaid.initialize({ startOnLoad: true });
</script>
```

File: packages/mermaid/src/docs/README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ It is a JavaScript based diagramming and charting tool that renders Markdown-ins
...
+[![Build Status](https://travis-ci.org/mermaid-js/mermaid.svg?branch=master)](https://travis-ci.org/mermaid-js/mermaid) [![NPM](https://img.shields.io/npm/v/mermaid)](https://www.npmjs.com/package/mermaid) [![Coverage Status](https://coveralls.io/repos/github/mermaid-js/mermaid/badge.svg?branch=master)](https://coveralls.io/github/mermaid-js/mermaid?branch=master) [![CDN Status](https://img.shields.io/jsdelivr/npm/hm/mermaid)](https://www.jsdelivr.com/package/npm/mermaid) [![NPM](https://img.shields.io/npm/dm/mermaid)](https://www.npmjs.com/package/mermaid) [![Join our Slack!](https://img.shields.io/static/v1?message=join%20chat&color=9cf&logo=slack&label=slack)](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE)
```
Comment (aloisklink) [inline packages/mermaid/src/docs/README.md:11]: This maybe should be another PR, but while we're here, we might as switch out the Travis CI badge [Build Status](https://travis-ci.org/mermaid-js/mermaid) for a GitHub Actions CI badge [Build CI Status](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml)

```suggestion
[![Build CI Status](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml/badge.svg)](https://github.com/mermaid-js/mermaid/actions/workflows/build.yml) [![NPM](https://img.shields.io/npm/v/mermaid)](https://www.npmjs.com/package/mermaid) [![Coverage Status](https://coveralls.io/repos/github/mermaid-js/mermaid/badge.svg?branch=master)](https://coveralls.io/github/mermaid-js/mermaid?branch=master) [![CDN Status](https://img.shields.io/jsdelivr/npm/hm/mermaid)](https://www.jsdelivr.com/package/npm/mermaid) [![NPM](https://img.shields.io/npm/dm/mermaid)](https://www.npmjs.com/package/mermaid) [![Join our Slack!](https://img.shields.io/static/v1?message=join%20chat&color=9cf&logo=slack&label=slack)](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE)
```

Also, we should probably change this in the mono-repo `README.md` too.

**Review by aloisklink — APPROVED** [review]
Looks good to me.

It might be better to:
- hardcode the URL as https://cdn.jsdelivr.net/npm/mermaid@9/dist/mermaid.min.js (e.g. use the latest `^9.0.0` version)
- maybe switch to recommending using ESM in the examples

But those are only suggestions :) Feel free to merge without them if you disagree or don't have time.

File: packages/mermaid/src/docs.mts (hunk 1 above)
Comment (sidharthv96) [reply packages/mermaid/src/docs.mts]: Done

File: vdocs/intro/index.md (hunk 2 above)
Comment (sidharthv96) [reply vdocs/intro/index.md]: I forgot the fact that we had an ESM build now. Was waiting for v10 release to update it. 🤣

File: packages/mermaid/src/docs/README.md (hunk 3 above)
Comment (sidharthv96) [reply packages/mermaid/src/docs/README.md]: Done

File: packages/mermaid/src/docs.mts [hunk 4]
```diff
@@ -41,6 +41,10 @@ import { remark } from 'remark';
 // @ts-ignore No typescript declaration file
 import flatmap from 'unist-util-flatmap';
 
+const version = (
```
Comment (aloisklink) [inline packages/mermaid/src/docs.mts:44]: Very minor, but it's potentially worth renaming this variable to `mermaidMajorVersion` or `MERMAID_MAJOR_VERSION`.

**Review by aloisklink — APPROVED** [review]
Looks good to me!

**Pull request merged**